prediction_market_agent/agents/microchain_agent/deploy.py

```diff
@@ -38,6 +38,7 @@ class DeployableMicrochainAgent(DeployableAgent):
     load_historical_prompt: bool = False
     system_prompt_choice: SystemPromptChoice = SystemPromptChoice.TRADING_AGENT
     task_description = AgentIdentifier.MICROCHAIN_AGENT_OMEN
+    description: str
```

Comment:
keeping the description in the class definition, as apposed to in the streamlit app